phaseshifts/wrappers.py

```
@@ -0,0 +1,733 @@
+            dst = os.path.abspath(
+                os.path.expanduser(os.path.expandvars("$CLEED_PHASE"))
+            )
+            EEASiSSSWrapper._copy_files(phsh_files, dst, verbose=True)
+
+        else:
+            EEASiSSSWrapper._copy_files(phsh_files, os.path.abspath("."), verbose=True)
+
+        return phsh_files
+
+
+class BVHWrapper(object):
```

Ruff (0.14.10)

Class `BVHWrapper` inherits from `object`

Remove `object` inheritance

(UP004)